modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java

```diff
@@ -78,7 +81,8 @@
 {
     private static final Logger LOG = LoggerFactory.getLogger( DumpServiceImpl.class );
 
-    private static final RepositoryId AUDIT_LOG_REPO_ID = RepositoryId.from( "system.auditlog" );
+    private static final Set<RepositoryId> SYSTEM_REPO_IDS =
```

Comment:
it is possible to generify with `RepositoryEntryService.findRepositoryEntryIds` and filter repos that start with `system.`

Reply:
Do we prohibit the creation of custom repos with `system.*` names?

Reply:
Actually, yes. Or we won't be able to freely use it ourselves.
Anyone who creates system.* steps into muddy waters.